pkgs/development/python-modules/ofxparse/default.nix

```diff
@@ -1,18 +1,22 @@
 { lib
 , buildPythonPackage
-, fetchPypi
+, fetchFromGitHub
 , six
 , beautifulsoup4
 , lxml
 }:
 
 buildPythonPackage rec {
   pname = "ofxparse";
-  version = "0.20";
+  version = "0.20-3236cf";
```

Comment:
I think we could instead define (or override) the required ofxparse in the ledger-autosync expression in order to expose a released version.

Reply:
fixed

Reply:
I'd like to offer a counter point: reevaluating the `pythonPackages` fixpoint as is needed here for overrides is expensive and should be avoided. If other packages should not depend on the unstable version of ofxparse, consider adding an ofxparse-unstable attribute to pythonPackages temporarily instead.

Reply:
@sternenseemann hm ok. I don't have any idea of the cost of a fixpoint evaluation:/
However, you don't need to override the `pythonPackages` set: you could use your `ofxparse` directly in the `propagatedBuildInputs` list (i assume `ofxparse` is only a direct dependency of `ledger-autosync`).

Since this `ofxparse` package is only used for this application, i don't think it's pertinent to expose it. Currently, there is no `*-unstable` packages in the Python package set. 

@sternenseemann would you agree with this?

Reply:
I tried adding `ofxparse` to the `propagatedBuildInputs` but I got an error message telling me `ofxparse` appears twice in the dependency tree of ledger-autosync.

Reply:
Due to `ofxclient` probably. I'd be in favor of just updating `ofxparse` globally or applying the necessary patches.

Reply:
I went back to my original change: modify ofxparse globally.